src/modules/controllers/task.controller.js

```diff
@@ -0,0 +1,102 @@
+const Task = require('../../models/task');
+
+const getAllTasks = async (req, res) => {
+  try {
+    const result = await Task.find();
+    res.status(200).send({ data: result })
+  } catch (error) {
+    res.status(400).send('Bad Request tasks not found')
+    console.error('tasks not found');
+  }
+}
+
+const createNewTask = async (req, res) => {
+  try {
+    const task = new Task({
+      text: req.body.text,
+      isCheck: false
+    });
+    const result = await task.save();
+    res.status(201).send(result);
+  } catch (error) {
+    res.status(400).send('Bad Request task not saved')
+    console.error('task not saved');
+  }
+};
+
+const changeTaskInfo = async (req, res) => {
+  try {
+    if (!req.params.hasOwnProperty('task_id')
+      || req.params.task_id === ''
+    ) {
+      throw new Error;
```

Comment:
Write default error
for example:
`throw new Error('Can't get task id')`
fix it everywhere